Update nano.md (#3771)
tldr-pages/tldr: pages/common/nano.md

# nano

> Simple, easy to use command-line text editor. An enhanced, free Pico clone.
> More information: <https://nano-editor.org>.

> - Open a specific file:

Reviewer: ```suggestion
- Open a file:
```

Reviewer 2: I don't think this is an improvement, it's correct to say that the file is the one specified, and not just "a file".

Reviewer: Okay, I edited it.

Reviewer 2: Still no improvement though. "Open a specific file" sounds better to me than "Open the specified file". It describes the general operation, using "a" instead of "the". We use "a specific" a lot more than "the specified" through all pages.

Reviewer: Perhaps you're right. I judged this according to some recent edits, so my view is probably skewed.

> `nano {{path/to/file}}`

Reviewer: ```suggestion
`nano {{/path/to/file}}`
```

Reviewer 2: Same here.

- Open a file positioning the cursor at the specified line and column:

> `nano +{{line}},{{column}} {{path/to/file}}`

Reviewer: ```suggestion
`nano +{{line}},{{column}} {{/path/to/file}}`
```

Reviewer 2: This should not be added. The path does not need to be absolute and this seems to imply it.

> > Simple, easy to use editor. An enhanced, free Pico clone.

Reviewer 2: ```suggestion
> Simple, easy to use command-line text editor. An enhanced, free Pico clone.
```

> More information: <https://nano-editor.org>.

> - Start nano in terminal with a specified filename:

Reviewer 2: ```suggestion
- Start nano opening a specific file:
```

Reviewer 3: Thinking about this, it probably isn't necessary to mention nano in each example. So they could probably just be:

- Open a specific file:
- Open a file with the cursor at a specific line:
- etc.

Reviewer 2: @pxgamer true, updated.

> `nano {{filename}}`

Reviewer 2: ```suggestion
`nano {{path/to/file}}`
```

> - Start nano in terminal with filename at specified line and column:

Reviewer 2: ```suggestion
- Start nano opening a file and positioning the cursor at the specified line and column:
```

Reviewer: `- Open a file starting at the specified line and column:`

Perhaps this could be more clear, but I'm unsure. The suggestion you made is quite long.

Reviewer 2: I don't think that's too long. It's medium sized. Just "starting at the specified line and column" is strangely worded, the command doesn't "start" anything.

> `nano +{{line}},{{column}} {{filename}}`

Reviewer 2: ```suggestion
`nano +{{line}},{{column}} {{path/to/file}}`
```

`nano {{filename}}`

> - Start nano in terminal with {filename} at line {line}, column {column}:

Reviewer 4: Let`s remove these curly brackets from the example description. 
Having them only on the example is enough because the example itself makes it pretty intuitive.

Author: NP, done (please note there are curly brackets on first example, as well)

`nano {{filename}}`

> - Start nano in terminal with filename at line,column:

Reviewer: Perhaps `at specified line and column` will read better?

Reviewer 4: I agree with @zdroid's suggestion.
